docs/01-app/01-getting-started/08-server-and-client-components.mdx

```diff
@@ -563,4 +559,26 @@ export async function getData() {
 
 Now, if you try to import the module into a Client Component, there will be a build-time error.
 
-> **Good to know**: The corresponding [`client-only` package](https://www.npmjs.com/package/client-only) can be used to mark modules that contain client-only logic like code that accesses the `window` object.
+The corresponding [`client-only` package](https://www.npmjs.com/package/client-only) can be used to mark modules that contain client-only logic like code that accesses the `window` object.
+
+In Next.js, installing `server-only`, or `client-only` is **optional**. Linting rules that prevent usage of extraneous dependencies, not declared in `package.json` may raise an issue if `server-only` or `client-only`, are not installed.
```

Comment:
```suggestion
In Next.js, installing `server-only` or `client-only` is **optional**. However, if your linting rules flag extraneous dependencies, you may install them to avoid issues.
```